dotnetv3/ECS/ECSActions/ECSWrapper.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.ECS;
+using Amazon.ECS.Model;
+using Microsoft.Extensions.Logging;
+
+namespace ECSActions;
+
+public class ECSWrapper
+{
+
+    private readonly AmazonECSClient _ecsClient;
+    private readonly ILogger<ECSWrapper> _logger;
+
+    /// <summary>
+    /// Constructor for the CloudWatch wrapper.
+    /// </summary>
+    /// <param name="ecsClient">The injected ECS client.</param>
+    /// <param name="logger">The injected logger for the wrapper.</param>
+    public ECSWrapper(AmazonECSClient ecsClient, ILogger<ECSWrapper> logger)
+
+    {
+        _logger = logger;
+        _ecsClient = ecsClient;
+    }
+
+    /// <summary>
+    /// List cluster ARNs available.
+    /// </summary>
+    /// <returns>The ARN list of clusters.</returns>
+    public async Task<List<string>> GetClusterARNSAsync()
+    {
+
+        Console.WriteLine("Getting a list of all the clusters in your AWS account...");
+        List<string> clusterArnList = new List<string>();
+        // Get a list of all the clusters in your AWS account
+        try
+        {
+            var listClustersResponse = await _ecsClient.ListClustersAsync(new ListClustersRequest
```

Comment:
If a list operation is intended to get all items, check the client to see if there is a paginator. Paginators are the recommended method for these list operations. https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/paginators.html
For example, you would use _ecsClient.Paginators.ListClusters here, and similarly apply the paginators instead of the do-while in the list operations.

Reply:
Thanks for your feedback! I was not aware of the Paginators class.